### DISCUSSION TITLE
What is the current status for doorstop 4 support?

### DISCUSSION THREAD

**Discussion by AevumTacet** [BepInEx/BepInEx#487] [BepInEx 6 proposals]
As simple as the title says, I wonder if there is still support for doorstop 4, if it was already successfully implemented in the Bepinex 6 pre.1 version or what are the plans for the future, in case it has not been implemented yet. Thank you for your time.

**Comment by ghorsington** [comment]
BepInEx 6.0.0-pre.1 is the first and last version before Doorstop 4 integration. If you need to use Doorstop 4, you can use the latest Bleeding Edge builds.

**Comment by AevumTacet** [comment]
The latest bleeding edge builds just do not load on ARM64 anymore. Actually, the only version of BepInEx that worked on ARM64, even loading some basic plugins, was an artifact, the Build PR #47, from the feature-doorstop-4 branch that was merged with the master branch.
